docs/website/versioned_docs/version-3.0.0/command-reference/registry.md

```diff
@@ -0,0 +1,107 @@
+---
+title: odo registry
+sidebar_position: 1
+---
+
+The `odo registry` command lists all the Devfile stacks from Devfile registries.
+
+The Devfile registries that are taken into account are the registries added with the command
+`odo preference registry`.
+
+## Available Flags
+
+By default, `odo registry` lists all the Devfile stacks from all the Devfile registries.
+
+These flags let you filter the listed Devfile stacks:
+
+* `--devfile <name>` to list the Devfile stacks with this exact name
+* `--devfile-registry <name>` to list the Devfile stack of this registry (this is the `name` used
+when adding the registry to the preferences with `odo preference registry add <name> <url>`)
+* `--filter <term>` to list the Devfile for which the term is found in the devfile name or description
```

Comment:
Can you also update the CLI Help message of this command? Currently, it says that the filter is based on the name of the component:

```
❯ odo help registry        
List all components from the Devfile registry

Usage:
  odo registry [flags]

Flags:
      --details                   Show details of each component
      --devfile string            Only the specific Devfile component
      --devfile-registry string   Only show components from the specific Devfile registry
      --filter string             Filter based on the name of the component
```